### DISCUSSION TITLE
EconomyShopGUI as a price provider

### DISCUSSION THREAD

**Discussion by kacmucha** [BG-Software-LLC/SuperiorSkyblock2#1700] [Pending Feature Requests] [answered]
As in Title. Please add EconomyShopGUI plugin as a price provider. One plugin is not enough. Not everyone uses the ShopGuiPlus plugin.

**Comment by Lemvs** [comment, marked as answer]
Added in latest dev build.

@OmerBenGera you can mark this answered and close it.
